Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:bionic as source
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY . selfoss/
+
```

Comment:
I would remove  this whole part and use `COPY . selfoss/` in the `composer` stage.

Reply:
> Hmm, seems good. going to update after testing whether it breaks something or not.

> I would remove this whole part and use `COPY . selfoss/` in the `composer` stage.

Reply:
> I would remove this whole part and use `COPY . selfoss/` in the `composer` stage.

It will reduce the build timing as well.